doc/conference_posters.md

```diff
@@ -9,7 +9,7 @@ There you can find powerpoint templates and logos of the main institutions and f
 
 ## Affiliations
 
-see [here](doc/papers.md#affiliation-for-authors-of-the-cpp-lab)
+see [here](https://cpp-lln-lab.github.io/CPP_LabGuide/papers/#affiliation-for-authors-of-the-cpp-lab)
```

Comment:
this should work, merging so thaat the github page is rendered and we can check online

```suggestion
see [here](papers.md#affiliation-for-authors-of-the-cpp-lab)
```